frontend/src/modules/user/views/loginView.js

```diff
@@ -30,6 +31,21 @@ define(function(require) {
       e && e.preventDefault();
       Origin.router.navigateToHome();
     },
+    
+    togglePassword: function(e) {
+      e && e.preventDefault();
+      
+      var passwordField = document.querySelector('.reveal-input-password');
```

Comment:
In adapt this type of selector isn't common, we tend to use jQuery selectors, please could you update this

```
var passwordField = this.$('.reveal-input-password');
var passwordReveal = this.$('.reveal-password-button');
```

You'll have to use attr like so

`passwordField.attr('type', 'text');`